contrib/babelfishpg_tsql/src/fts_scan.l

```diff
@@ -21,23 +21,28 @@ static YY_BUFFER_STATE scanbufhandle;
 
 %%
 
-"AND"           { yylval = "AND"; yyerror(NULL, "Full-text search conditions with boolean operators are not currently supported in Babelfish"); return AND_TOKEN; }
-"AND NOT"       { yylval = "AND NOT"; yyerror(NULL, "Full-text search conditions with boolean operators are not currently supported in Babelfish"); return AND_NOT_TOKEN; }
-"OR"            { yylval = "OR"; yyerror(NULL, "Full-text search conditions with boolean operators are not currently supported in Babelfish"); return OR_TOKEN; }
-"NOT"			{ yylval = "NOT"; yyerror(NULL, "Full-text search conditions with boolean operators are not currently supported in Babelfish"); return NOT_TOKEN; }
+"AND"           { yylval = "AND"; return AND_TOKEN; }
+"and"           { yylval = "AND"; return AND_TOKEN; }
+"&"           	{ yylval = "AND"; return AND_TOKEN; }
+"AND NOT"       { yylval = "AND NOT"; return AND_NOT_TOKEN; }
```

Comment:
Can you please verify if this will also work when there are more than one space in between in AND and NOT?